docs/rules/generator-star-spacing.md

```diff
@@ -71,7 +71,11 @@ And the option has shorthand as a string keyword:
 "generator-star-spacing": ["error", "after"]
 ```
 
-When using `{"before": true, "after": false}` this placement will be enforced:
+## Examples
```

Comment:
Instead of h2 Examples, how about insert h2 **Options** which is definitely missing before the sentence:

The rule takes one option